recipes/gperftools/run_test.sh

```diff
@@ -0,0 +1 @@
+#!/bin/sh
```

Comment:
would be nice to get a real test. if not then i think this file can just be deleted

Reply:
I used this file to pass on conda-forge-linter test. I added `make check` to do the initial test. Should be sufficient `make check`? Actually I don't know much about `gperftools` to make its tests. I am packaging `gperftools` because it is a requirement for another package.

Reply:
make check is probably fine

Reply:
Typically the most robust tests for compiled code are in the build. Though we like to have something to verify the package is as expected. We usually go for [file tests]( https://conda-forge.org/docs/testing.html ), which are pretty easy to add.